### PR TITLE
Batch 2 of bug doc text

### PR DESCRIPTION
**Pull request by opayne1**
Adds bug fixes to bare metal, build, and management console sections

http://file.rdu.redhat.com/opayne/RN-batch-2/release_notes/ocp-4-11-release-notes.html#ocp-4-11-bug-fixes

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -937,10 +937,20 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-bare-metal-hardware-bug-fixes"]
 ==== Bare Metal Hardware Provisioning
 
+* Previously, when writing the RHCOS image onto some disks, the qemu-img was allocating space onto the entire disk, including sparse areas. This extended the time of the writing process on some hardware. This update disables the qemuimg sparse in image creation. As a result, the image writing would no longer take a long time on effected hardware. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002009[*BZ#2002009*])
```
Comment (mburke5678) [inline release_notes/ocp-4-11-release-notes.adoc:940]: s/effected/affected

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -937,10 +937,20 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
+* Previously, if the “rotational” field was set for `RootDeviceHints`, the host could fail the provision. With this update, the properky copies and checks the “rotational” field in RootDeviceHints. As a result, the provisioning will succeed when using the “rotational” field. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2053721[*BZ#2053721*])
```
Comment (mburke5678) [inline release_notes/ocp-4-11-release-notes.adoc:942]: s/the properky/the installer properly

@opayne1 Please check with the doc text author on this one. Not sure if I have the fix right. But, _properky_ is certainly wrong.

Comment (mburke5678) [reply release_notes/ocp-4-11-release-notes.adoc:942]: s/"rotational"/`rotational` in three places

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -937,10 +937,20 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
 [discrete]
 [id="ocp-4-11-builds-bug-fixes"]
 ==== Builds
 
+* Previously, using a forward slash (`/`) in the `ImageLabel` name of a `BuildConfig` instance resulted in an error. This fix resolves the issue by changing the utility used for validation. As a result, you can use a forward slash in the `ImageLabel` name of a `BuildConfig` instance. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2105167[*BZ#2105167*])
+
+* Previously, when using the `$ oc new-app --search <image_stream_name>` command, you could receive an incorrect message related to `docker.io` images. This resulted in confusion because {product-title} should not be using image streams that point to `docker.io`. This fix adds code checks to prevent the reference to 'docker.io'. As a result, the output from that command does not include the message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049889[*BZ#2049889*])
```
Comment (mburke5678) [inline release_notes/ocp-4-11-release-notes.adoc:950]: s/confusion/confusion for the user

Sorry. I worked this one.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (mburke5678) [reply release_notes/ocp-4-11-release-notes.adoc:942]: s/will succeed/succeeds

File: release_notes/ocp-4-11-release-notes.adoc [hunk 4]
```diff
@@ -937,10 +937,20 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
+* Previously, Shared Resource CSI Driver metrics were not exported to the Telemetry service. As a result, usage metrics for the Shared Resource CSI Driver could not be analyzed. With this fix, Shared Resource CSI Driver metrics are exposed to Red Hat telemetry. As a result, usage metrics for the Shared Resource CSI Driver can be collected and analyzed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2058225[*BZ#2058225*])
```
Comment (mburke5678) [inline release_notes/ocp-4-11-release-notes.adoc:952]: s/Red Hat telemetry/the Telemetry.service

Sorry again! Sloppy work on my part.

**Comment by mburke5678** [comment]
@opayne1 A few nits. Otherwise LGTM

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:942]: looks like it properky was a typo in the doc text in the BZ and transferred over. I have updated.

**Pull request merged**